docs/protocol/index.mdx

```diff
@@ -39,3 +39,28 @@ Dive into any of the sections below to learn more about the protocol's inner wor
 />
 
 </Cards>
+
+## Blockchain scaling with IPC
```

Comment:
i added this IPC callout at the start of the protocol section. i tried to be brief yet pull the primary parts from IPC intro docs pages